File: app/src/main/java/br/com/ricardo/testeandroid/view/MessageFragment.java
package br.com.ricardo.testeandroid.view;

import android.graphics.Typeface;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import br.com.ricardo.testeandroid.R;


public class MessageFragment extends Fragment {

    private TextView textMessageTitle;
    private TextView textMessageSubtitle;
    private TextView textMessageLink;

    private Typeface font;

    private FragmentManager fragmentManagerMessage;
    private FragmentTransaction fragmentTransactionMessage;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {

        View messageView = inflater.inflate(R.layout.fragment_message, container, false);

        textMessageTitle = (TextView) messageView.findViewById(R.id.message_title);
        textMessageSubtitle = (TextView) messageView.findViewById(R.id.message_subtitle);
        textMessageLink = (TextView) messageView.findViewById(R.id.message_link);

        font = Typeface.createFromAsset(getActivity().getAssets(), "font/DINPro-Medium.otf");
        textMessageTitle.setTypeface(font);
        textMessageSubtitle.setTypeface(font);
        textMessageLink.setTypeface(font);

        textMessageLink.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                fragmentManagerMessage = getActivity().getSupportFragmentManager();
                fragmentTransactionMessage = fragmentManagerMessage.beginTransaction();
                fragmentTransactionMessage.replace(R.id.main_frame_container, new ContactFragment());
                fragmentTransactionMessage.commit();

            }
        });


        return messageView;

    }
}

File: app/src/main/java/com/luccasmelo/santandereveris/ui/fragment/MessageFragment.kt
package com.luccasmelo.santandereveris.ui.fragment

import androidx.databinding.DataBindingUtil
import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.navigation.fragment.findNavController
import com.luccasmelo.santandereveris.R
import com.luccasmelo.santandereveris.databinding.InvestmentFragmentBinding
import com.luccasmelo.santandereveris.databinding.MessageFragmentBinding

class MessageFragment: Fragment(){
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val binding: MessageFragmentBinding = DataBindingUtil.inflate(inflater, R.layout.message_fragment,container,false)
        binding.txtClickContact.setOnClickListener{
            findNavController().navigate(R.id.action_contact)
        }
        return binding.root
    }
}